forest-node/ansible/roles/forest/tasks/main.yml

```diff
@@ -0,0 +1,62 @@
+---
+# tasks file for role/forest
+- name: Upgrade all apt packages
+  apt: upgrade=dist force_apt_get=yes
+
+- name: Create a login user
+  user:
+    name: "{{ user }}"
+    state: present
+
+- name: Add public key to authorized_keys
+  authorized_key:
+    user: "{{ user }}"
+    state: present
+    key: "{{ lookup('file', '~/.ssh/id_rsa.pub') }}"    # assuming the name of the public key file is `id_rsa.pub`
+
+- name: Allow specific users to log in
+  ansible.builtin.lineinfile:
+    dest: /etc/ssh/sshd_config
+    regexp: '^AllowUsers'
+    line: 'AllowUsers {{ user }}'
+    state: present
```

Comment:
Is this necessary?

Reply:
Just a best practice is to allow just a specific user to interact with the server - basic security play

Reply:
Unless I'm mistaken, the Digital Ocean image we're using already allows users to connect via SSH. Are you saying this is not the case?

Reply:
It is still `ssh` being used here; we are just using a new user to log in to the server and blocking root access.

Reply:
For sure, yes and allow only the specified user